README.md

```diff
@@ -1,5 +1,14 @@
 # Arbitrum Governor Upgrade
 
+## New Address Registry Contract
+
+After the upgrade, a new address registry contract can be deployed by running the following command in the [Arbitrum Governance](https://github.com/ArbitrumFoundation/governance) repo:
+
+```bash
+make
+forge create src/gov-action-contracts/address-registries/L2AddressRegistry.sol:L2AddressRegistry --constructor-args <NEW_CORE_GOVERNOR_CONTRACT_ADDRESS> <NEW_TREASURY_GOVERNOR_CONTRACT_ADDRESS> 0xF3FC178157fb3c87548bAA86F9d24BA38E649B58 0x1D62fFeB72e4c360CcBbacf7c965153b00260417 --rpc-url <ARB_ONE_RPC_URL> --private-key <PRIVATE_KEY>
```

Comment:
It is probably best not to include the private key in the command. It would probably be better to use the interactive flag.

Reply:
Do you have a sample command we could test with? Or would I have to deploy a governor and treasury contract first?

Reply:
`forge create src/gov-action-contracts/address-registries/L2AddressRegistry.sol:L2AddressRegistry --constructor-args 0xf07DeD9dC292157749B6Fd268E37DF6EA38395B9 0x789fC99093B09aD01C34DC7251D0C89ce743e5a4 0xF3FC178157fb3c87548bAA86F9d24BA38E649B58 0x1D62fFeB72e4c360CcBbacf7c965153b00260417 --rpc-url https://arbitrum.llamarpc.com --interactive` But this would actually deploy to Arb mainnet. There are some relational conditions between these contracts that need to be met, so I'm not sure how to test it without the properly setup testnet contract addresses.

Reply:
I feel like there is a way to look for contracts with the bytecode deployed on other networks using etherscan? That way maybe we can test on arbitrum sepolia. But can't seem to find that option right now. Any recs?

Reply:
[b624bdd](https://github.com/ScopeLift/arbitrum-governor-upgrade/pull/30/commits/b624bdd5d88a536ae840e29e5a50058815b176b0)